### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by elight**
Fixed a typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ list who has access to an application
 
 
 ```bash
-$ heroku acesss:update EMAIL [--app APP]
+$ heroku accesss:update EMAIL [--app APP]
```
Comment (raulb) [inline README.md:155]: @elight Thanks!, shouldn't it be `access` instead? :P

Comment (raulb) [inline README.md:155]: Just realized I had a double typo there, hahaha

**Comment by elight** [comment]
Lol oy vey I missed that!

**Comment by raulb** [comment]
@elight me too! :)

**Pull request merged**